lib/alchemy/upgrader/tasks/cells_upgrader.rb

```diff
@@ -94,31 +97,50 @@ def move_cell_views
         Dir.glob("#{cells_view_folder}/*").each do |view|
           filename = File.basename(view).gsub(/(_\w+)\.(\w*\.)?(erb|haml|slim)/, '\1_view.\2\3')
           FileUtils.mv(view, "#{elements_view_folder}/#{filename}")
-          puts "Moved #{File.basename(view)} into `app/views/alchemy/elements/` folder"
+          puts "   Moved #{File.basename(view)} into `app/views/alchemy/elements/` folder"
         end
         FileUtils.rm_rf(cells_view_folder)
       else
         puts "No cell views found. Skip"
       end
     end
 
+    def generate_editor_partials
+      puts "-- Generate editor partials"
+      Rails::Generators.invoke('alchemy:elements', ['--skip'])
+    end
+
     def update_cell_views
       if Dir.exist? cells_view_folder
         puts "-- Update cell views"
         Dir.glob("#{cells_view_folder}/*").each do |view|
-          gsub_file(view, /cell\.elements/, 'cell.nested_elements')
+          gsub_file(view, /cell\.elements/, 'element.nested_elements.published')
+          gsub_file(view, /render_elements\(?from_cell:\scell\)?/, 'render element.nested_elements.published')
+          gsub_file(view, /cell/, 'element')
         end
       else
         puts "No cell views found. Skip"
       end
     end
 
+    def update_render_cell_calls
+      puts "-- Update render_cell calls"
+      Dir.glob("#{alchemy_views_folder}/**/*").each do |view|
+        next if File.directory?(view)
+        gsub_file(view, /render_cell[\(\s]?([:'"]?[a-z_]+['"]?)\)?/, 'render_elements(only: \1, fixed: true)')
```

Comment:
Was it previously supported to use a symbol here instead of a string? Should that be supported in the regex?

Reply:
Symbols are supported by the regex

```rb
>> r = /render_cell[\(\s]?([:'"]?[a-z_]+['"]?)\)?/
=> /render_cell[\(\s]?([:'"]?[a-z_]+['"]?)\)?/
>> "render_cell :left".match r
=> #<MatchData "render_cell :left" 1:":left">
```